browser/directives/pathValidator.js

```diff
@@ -13,10 +14,14 @@ function pathValidator() {
 
       function validateFormatWindows(value) {
         let trimmedValue = value.trim();
-
         return pathWindowsRegex.test(trimmedValue);
       }
 
+      function validateAscii(value) {
+        let trimmedValue = value.trim();
```

Comment:
Trimming is not required, because angular automatically trims text fields, unless explicitly said 'do not trim' through ngTrim directive. See details here https://docs.angularjs.org/api/ng/input/input%5Btext%5D.